src/cli-options.ts

```diff
@@ -111,6 +111,23 @@ npm run test
 Saving partially upgraded package.json
 `
 
+/** Extended help for the filterMeta option. */
+const extendedHelpFilterMeta = (): string => {
+  return `Filter packages after the metadata is fetched. Only available in .ncurc.js or when importing npm-check-updates as a module:
+
+  ${chalk.gray(`/**
+    @param packument        The object returned from NPM.
+    @returns                Boolean value, true to include the dependency in the output, false to exclude it.
+  */`)}
+  ${chalk.cyan('filterMeta')}: (packument} {
+    if (packument.name === 'typescript') {
+      return true
+    }
+    return return false
+  }
```

Comment:
Simpler: `packument => packument.name === 'typescript'`.

However, I don't think `packument.name` is the best example to use here since it is available pre-fetch. There would be no reason to filter on the name in this way. Instead, let's give an example of filtering on version like your use case.